### DISCUSSION TITLE
Move and renaming without breaking seeding

### DISCUSSION THREAD

**Discussion by javikobcn** [Flexget/Flexget#3584] [Q&A]
Hi, i need help to keep seeding my torrents after move and rename my files like the title says.
Using transmission i have discovered when i move/rename a file to another different folder of /downloads all keep seeding ok but after reboot transmission container or reboot the nas, all seeding files stops and it's impossible to "reseed" again.
I think i need a proper config to solve this but im pretty noob to do it...

**Comment by gazpachoking** [comment]
If you want flexget to initiate the move, you should be able to use `from_transmission` plugin to get the currently loaded torrents in transmission, make sure you accept the ones you want to move, then use `transmission` plugin as the output, and set the `path` setting to the new place you want them. This will cause transmission to do the moving of the file, and it will keep track of the new location that it should seed the files from.

**Reply by gazpachoking** [reply]
If you mean you are moving the files manually, you should be able to do that directly from transmission itself.

**Reply by javikobcn** [reply]
> If you want flexget to initiate the move, you should be able to use `from_transmission` plugin to get the currently loaded torrents in transmission, make sure you accept the ones you want to move, then use `transmission` plugin as the output, and set the `path` setting to the new place you want them. This will cause transmission to do the moving of the file, and it will keep track of the new location that it should seed the files from.

I want it to be fully automatic, there is my config, as you can see now i have 2 task for rename and moving to other directories, how can implement it to keep seeding at the new directories?

https://hastebin.com/piqugatido.yaml

**Comment by javikobcn** [comment]
I keep trying to do it but im having problems in every try. What is wrong there?
https://hastebin.com/ewukeconep.yaml  
Here is the log after executing the config
https://hastebin.com/kamofebura.md 
As you can see, it says the task accept and move the files matched like movies to the required path but the reality is the file keeps in the transmission download torrents folder without moving....
And the filename didn't change

**Comment by ProteinPig** [comment]
The mathed you do it manually can not reseed correctly after you rename the file. You just look like you are reseed, because of no leecher download from you. And some trackers consider that as cheat. Because you don't have the right file (you have rename it).

I suppose you use symlink plugin to rename  the "new file", and with symlink you don't need double disk space.
